src/main/docker/Dockerfile.jvm

```diff
@@ -21,34 +21,36 @@
 # docker run -i --rm -p 8080:8080 -p 5005:5005 -e JAVA_ENABLE_DEBUG="true" quarkus/release-manager-jvm
 #
 ###
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.3
 
 ARG JAVA_PACKAGE=java-11-openjdk-headless
 ARG RUN_JAVA_VERSION=1.3.8
 
 ENV LANG='en_US.UTF-8' LANGUAGE='en_US:en'
 
 # Install java and the run-java script
-# Also set up permissions for user `1000`
+# Also set up permissions for user `1001`
 RUN microdnf install curl wget ca-certificates ${JAVA_PACKAGE} \
     && microdnf update \
     && microdnf clean all \
     && mkdir /deployments \
-    && chown 1000 /deployments \
+    && chown 1001 /deployments \
     && chmod "g+rwX" /deployments \
-    && chown 1000:root /deployments \
+    && chown 1001:root /deployments \
     && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
-    && chown 1000 /deployments/run-java.sh \
-    && chmod 755 /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 540 /deployments/run-java.sh \
     && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
 
 # Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-
-COPY target/lib/* /deployments/lib/
-COPY target/*-runner.jar /deployments/app.jar
+# We make four distinct layers so if there are application changes the library layers can be re-used
+COPY --chown=1001 target/quarkus-app/lib/ /deployments/lib/
+COPY --chown=1001 target/quarkus-app/*.jar /deployments/
+COPY --chown=1001 target/quarkus-app/app/ /deployments/app/
+COPY --chown=1001 target/quarkus-app/quarkus/ /deployments/quarkus/
 
 EXPOSE 8080
-USER 1000
+USER 1001
 
 #ENTRYPOINT [ "/deployments/run-java.sh" ]
```

Comment:
Should be `ENTRYPOINT [ "/deployments/run-java.sh" ] ` as we will use a different Dockerfile to build the image for jenkins jnlp.

Reply:
yes, let's keep these files as generated initially

Reply:
Can you then remove the `#` in front of the `ENTRYPOINT` ? @aureamunoz

Reply:
yes, it's done in the lasts commits, isn't it?